src/styles/_cards.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  font-size: $font-size;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  padding: 16px;
+  flex-direction: column;
+  align-items: center;
+  cursor: pointer;
+  transition: all 0.3s ease;
```

Comment:
please don't use `transition: all` - list all properties you want to use transition on instead